OpenCL.pc.in

```diff
@@ -0,0 +1,8 @@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
+
+Name: OpenCL
+Description: Khronos OpenCL ICD Loader
+Version: 3.0
+Libs: -L${libdir} -lOpenCL
+Cflags: -I${includedir}
```

Comment:
```suggestion
libdir=@CMAKE_INSTALL_FULL_LIBDIR@

Name: OpenCL
Description: Khronos OpenCL ICD Loader
Requires: OpenCL-Headers
Version: 3.0
Libs: -L${libdir} -lOpenCL
```
This would work in conjunction with: https://github.com/KhronosGroup/OpenCL-Headers/pull/213 that would need to be merged before adopting this change.

Reply:
Outdated